RubeesEat/RubeesEat.csproj

```diff
@@ -6,4 +6,11 @@
         <ImplicitUsings>enable</ImplicitUsings>
     </PropertyGroup>
 
+    <ItemGroup>
+      <Content Update="Dockerfile">
+        <ExcludeFromSingleFile>true</ExcludeFromSingleFile>
+        <CopyToPublishDirectory>PreserveNewest</CopyToPublishDirectory>
+      </Content>
+    </ItemGroup>
+

```

Comment:
I don't think we need this. The `Dockerfile` is not necessary for the project output and thus does not need to be marked as `Content`.